app/routes/products.$productSlug/route.tsx

```diff
@@ -44,8 +44,31 @@ export default function ProductDetailsPage() {
         getInitialSelectedOptions(product.productOptions)
     );
 
+    const getIsOutOfStock = () => {
```

Comment:
I know it is quite uncommon to add "get" to function if next word is "is", so I suggest to rename to `isOutOfStock`

![image](https://github.com/user-attachments/assets/cd599761-301f-4017-93e4-d2243818e64f)

Reply:
the line below there is `const isOutOfStock = getIsOutOfStock()` to avoid calling this function multiple times

Reply:
`outOfStock = isOutOfStock()` ?